run.sh

```diff
@@ -86,15 +86,16 @@ fi
 
 if [[ "$ALERT_MANAGER_URLS" != "" ]]; then
     sed -i -e 's;$ALERT_RULES_FILE;'"- $ALERT_RULES_FILE"';g' "/prometheus.yml.tmpl"
-
-    cat >> "/prometheus.yml.tmpl" <<- EOM
+	if ! grep -q alerting: "/prometheus.yml.tmpl"; then
+	cat >> "/prometheus.yml.tmpl" <<- EOM
```

Comment:
@KarineValenca, not sure what exactly this is doing. Could you give a brief explanation?

Reply:
I found out that the error of issue #30 is caused by a duplication in alerting configuration. So, this line checks if there is already a configuration for alerting in prometheus.yml.tmpl file.
It looks at the string "alerting:" inside the prometheus.yml.tmpl file, and if it finds nothing, it adds the alerting configuration into the file. Otherwise, if there is a string "alerting:" it means that the alerting configuration is already set, so it's not necessary to add the configuration again.

Reply:
Oh I see! Tricky bug this one! Thank you @KarineValenca !

Reply:
Just another question: If I setup a persistent storage for a `BB Promster` container and I change the `ALERT_MANAGER_URLS` value, wouldn't this code prevent me from getting the `prometheus.yml.tmpl` updated?

Reply:
I think you're right. I'm going to rethink the solution.